org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/GradleBuildFile.xtend

```diff
@@ -27,7 +27,11 @@ class GradleBuildFile extends TextFile {
 						compile project(':�p.name�')
 					�ENDFOR�
 					�FOR dep : mavenDependencies�
-						�dep.scope.gradleNotation� "�dep.groupId�:�dep.artifactId�:�dep.version�"
+						�IF dep.version !== null�
+							�dep.scope.gradleNotation� "�dep.groupId�:�dep.artifactId�:�dep.version�"
```

Comment:
wonder if we just add the if for the 3rd part.
would find it better readable

Reply:
Did you recognize that there is a difference in single vs double quote also?

Reply:
yes

Reply:
```
					«FOR dep : mavenDependencies»
						«dep.scope.gradleNotation» «IF dep.version !== null»"«dep.groupId»:«dep.artifactId»:«dep.version»"«ELSE»'«dep.groupId»:«dep.artifactId»'«ENDIF»
					«ENDFOR»
```
is actually less readable, or what did you mean?

Reply:
then lets go your way